Rocket.Chat/Models/Message/UnmanagedMessage.swift

```diff
@@ -53,6 +53,8 @@ struct UnmanagedMessage: UnmanagedObject, Equatable {
     var markedForDeletion: Bool
     var emoji: String?
     var avatar: String?
+    var snippetName: String?
+    var snippetId: String?
```

Comment:
Shouldn't you add these new fields to the == method?

Reply:
@cardoso I don't think this is necessary to be honest. This field will never change on the same message.

Reply:
:+1: